vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -92,93 +171,143 @@ $(document).ready(function() {
 
 });
 </script>
-
 <ol class="breadcrumb">
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/list" />">Modules</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
-  <li class="breadcrumb-item active">${slide.name}</li>
+    <li class="breadcrumb-item"><a
+        href="<c:url value="/staff/dashboard" />"
+    >Dashboard</a></li>
+    <li class="breadcrumb-item"><a
+        href="<c:url value="/staff/module/list" />"
+    >Modules</a></li>
+    <li class="breadcrumb-item"><a
+        href="<c:url value="/staff/module/${module.id}" />"
+    >${module.name}</a></li>
+    <li class="breadcrumb-item active">${slide.name}</li>
 </ol>
-
-<h1>Slide: ${slide.name}</h1>
+<div class="error"></div>
+<!-- title -->
+<div class="row align-items-center">
+    <h1 id="title" style="margin-bottom: 0%; margin-left: 1%;">Slide: ${slide.name}</h1>
+    <a id="editTitle" class="btn" href="#"
+        style="float: left; margin-right: 1%;"
+    ><i class="fas fa-edit"></i></a>
+    <button id="submitTitle" type="button"
+        class="btn btn-outline-success"
+        style="float: left; margin-right: 1%;"
+    >Save</button>
+</div>
 <div class="alert alert-light" role="alert">
-  Created on <span class="date">${slide.creationDate}</span> by ${slide.createdBy}.<br>
-  Modified on <span class="date">${slide.modificationDate}</span> by ${slide.modifiedBy}.
+    Created on <span class="date">${slide.creationDate}</span> by
+    ${slide.createdBy}.<br> Modified on <span class="date">${slide.modificationDate}</span>
+    by ${slide.modifiedBy}.
 </div>
-<h5>Description:</h5>
-<p>${slide.description}</p>
-	
-
-
-<nav class="navbar navbar-expand-sm navbar-light bg-light">
-<div class="dropdown">
-  <button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-    Add content
-  </button>
-  <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-    <a id="addText" class="dropdown-item" href="#">Add Text</a>
-    <a id="addImage" class="dropdown-item" href="#">Add Image</a>
-  </div>
+<!-- description -->
+<div style="margin-left: .1%;" class="row align-items-center">
+    <h5 style="margin-bottom: 0px;">Description:</h5>
+    <p id="description"
+        style="margin-top: 1%; margin-left: 1%; margin-bottom: .5rem;"
+    >${slide.description}</p>
+    <a id="editDescription" class="btn" href="#"
+        style="margin-right: 1%; font-size: .66rem; border-radius: .15rem; padding-top: .5%;"
+    ><i class="fas fa-edit"></i></a>
```

Comment:
let's put the edit icon behind the description label, it's a little odd where it is right now.